gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -2342,6 +2345,11 @@ <RequestT extends StorageRequest<?>> RequestT initializeRequest(
       String token = downscopedAccessTokenFn.apply(accessBoundaries);
       request.getRequestHeaders().setAuthorization("Bearer " + token);
     }
+    if (storageOptions.isGcsAuditLogEnabled()
+        && !storageOptions.getHttpRequestHeaders().containsKey(CUSTOM_AUDIT_USER_HEADER)) {
+      String user = getUgiUserName();
+      if (!user.isEmpty()) request.getRequestHeaders().set(CUSTOM_AUDIT_USER_HEADER, user);
```

Comment:
single line if statements can be avoided. It is error prone and harder to read.

Reply:
Done.